Source/SA/RFmx LTE/RFmxLTE.cs

```diff
@@ -201,6 +199,22 @@ public static void ConfigureModAcc(RFmxLteMX lte, ModAccConfiguration modAccConf
             lte.ModAcc.Configuration.ConfigureSynchronizationModeAndInterval(selectorString, modAccConfig.SynchronizationMode, modAccConfig.MeasurementOffset, modAccConfig.MeasurementLength);
             lte.ModAcc.Configuration.ConfigureEvmUnit(selectorString, modAccConfig.EvmUnit);
         }
+        public static void SelectAndInitiateMeasurements(RFmxLteMX lte, RFmxLteMXMeasurementTypes[] measurements, AutoLevelConfiguration autoLevelConfig,
+            bool enableTraces = false, string selectorString = "", string resultName = "")
+        {
+            // Aggregate the selected measurements into a single value
+            // OR of 0 and x equals x
+            RFmxLteMXMeasurementTypes selectedMeasurement = 0;

```

Comment:
```suggestion
            RFmxLteMXMeasurementTypes selectedMeasurements = 0;
```
suggestion for clarity, but won't request it to be changed